src/views/simulation/resultats-detail.vue

```diff
@@ -80,18 +80,17 @@ export default {
     },
   },
   mounted() {
-    if (this.mockResultsNeeded()) {
-      this.mock(this.$route.params.droitId)
+    if (this.mock(this.$route.params.benefitId)) {
       return
```

Comment:
```suggestion
    if (this.mockResultsNeeded()) {
      this.mock(this.$route.params.benefitId)
      return
```

J'ai modifié cette partie depuis et `this.mock` tout seul ne retourne plus rien